includes/_nav.htm

```diff
@@ -11,6 +11,12 @@
         <a class="nav__item nav__sub__item" href="/documentation#gettingStarted">getting started</a>
         <a class="nav__item nav__sub__item" href="/documentation#api">APIs</a>
         <a class="b-nav__streams nav__item nav__sub__item" href="/documentation/streams">kafka streams</a>
+        <!-- Streams child pages -->
```

Comment:
Could you paste a screenshot of this change?

Reply:
@guozhangwang here you go: 
![image](https://user-images.githubusercontent.com/11722533/31085984-7244dd88-a74d-11e7-919d-a3130ea918ac.png)